examples/hosts/iframe-host/src/outer.ts

```diff
@@ -83,25 +69,43 @@ async function getFluidObjectAndRender(container: Container, div: HTMLDivElement
     view.render(div, { display: "block" });
 }
 
-export async function loadDiv(divId: string) {
-    const div = document.getElementById(divId) as HTMLDivElement;
+async function loadOuterDataStoreAndLogDivs(
```

Comment:
this is wrong. this shouldn't reuse anything from the iframe case. it should be a completely standalone, and just use the normal flow to load in the div. it is simply for testing to have a second client

Reply:
misunderstood the intention of the previous comment which is also what lead to the IContainerProxy bloat.  should be a lot more minimal now.

---
In reply to: [530576670](https://github.com/microsoft/FluidFramework/pull/4404#discussion_r530576670) [](ancestors = 530576670)